### PR TITLE
Tests: devtools E2E test

### PR DESCRIPTION
**Pull request by bershanskiy**
This adds a test for Devtools UI which loads a dummy page (with some dummy styles), changes config (and checks that page styles changed), then resets styles (and checks that page styles went back to the original ones).

This test probably would have caught broken Devtools if we had it (I can check this if needed).

### CODE REVIEW

File: tests/browser/globals.d.ts [hunk 1]
```diff
@@ -10,6 +10,9 @@ declare global {
     const popupUtils: {
         click: (selector: string) => Promise<void>;
         exists: (selector: string) => Promise<boolean>;
-        getBoundingRect: (selector: string) => Promise<{left: number; top: number; width: number; height: number}>;
```
Comment (bershanskiy) [inline tests/browser/globals.d.ts:13]: I removed `popupUtils.getBoundingRect` because it is not used.

File: tests/browser/environment.js [hunk 2]
```diff
@@ -174,7 +185,7 @@ class PuppeteerEnvironment extends JestNodeEnvironment {
                 ws.on('close', () => sockets.delete(ws));
             });
 
-            function sendToPopup(message) {
```
Comment (bershanskiy) [inline tests/browser/environment.js:177]: `sendToPopup()` currently broadcasts every message to every socket for simplicity. I don't think it's worth making it smarter yet.

File: tests/browser/globals.d.ts (hunk 1 above)
Comment (Gusted) [reply tests/browser/globals.d.ts:13]: Seems good.

File: tests/browser/environment.js (hunk 2 above)
Comment (Gusted) [reply tests/browser/environment.js:177]: If it works, don't touch it ;)

**Review by Gusted — APPROVED** [review]
First glance seems to be fine for me. No weird code shenanigans. Nice to have a E2E for Dev tools.

**Pull request merged**